tap_google_sheets/client.py

```diff
@@ -213,7 +213,8 @@ def get_access_token(self):
     @backoff.on_exception(backoff.expo,
                           (Server5xxError, ConnectionError, Server429Error),
                           max_tries=7,
-                          factor=3)
+                          factor=3,
+                          jitter=None)
```

Comment:
what is the purpose of this jitter=None

Reply:
As we are now making 2 API calls, for bulk requests like 5k or 10k records, google-sheets throws [Quota/Rate limit exceeded](https://developers.google.com/sheets/api/limits#:~:text=If%20you%20exceed%20a%20quota,you%20can%20make%20per%20day.). Hence to ensure that the tap completes the retires in more than a minute, we have introduced the jitter=None

Reply:
@namrata270998 You meant to say retries I believe

Reply:
Yeah sorry it was a typo, I meant retries